anisoap/representations/ellipsoidal_density_projection.py

```diff
@@ -120,7 +120,7 @@ def pairwise_ellip_expansion(
                     )
 
                     moments = compute_moments_inefficient_implementation(
-                        precision, center, maxdeg=lmax + np.max(num_ns)
+                        precision, center, maxdeg=maxdeg
```

Comment:
looks like another remnant of future PRs

Reply:
this is not a remnant, it's necessary. Having line 78 and line 123 will ensure together makes it so that we always calculate the right number of moments.

Reply:
:+1: